src/index.ts

```diff
@@ -46,6 +46,14 @@ export function setInputStabilityCheckEnabled(enabled: StabilityCheck) {
   globalStabilityCheck = enabled
 }
 
+function getTime() {
```

Comment:
I am wondering if wouldn't be better if definition of `getTime` would look like:
```typescript
const getTime = performance && typeof performance.now === 'function'
  ? performance.now.bind(performance)
  : Date.now.bind(Date)
```

In that case no extra conditions or jumps will be done while the function executes.